test/unit/definitions/divorce/AuthorisationCaseField.test.js

```diff
@@ -19,14 +19,18 @@ describe('AuthorisationCaseField', () => {
         'AuthorisationCaseField-deemed-and-dispensed-nonprod',
         'AuthorisationCaseField-general-email-nonprod',
         'AuthorisationCaseField-general-referral-nonprod',
+        'AuthorisationCaseField-alt-service-process-server-nonprod',
```

Comment:
can you please use alphabetical order here?

Reply:
yup